File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: list[int]) -> int:
        count = 1

        for i in range(len(nums) - 1):
            if nums[i] != nums[i + 1]:
                nums[count] = nums[i + 1]
                count += 1

        return count

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.c
int removeDuplicates(int* nums, int numsSize) {
    int i = 0;
    int j = 1;
    int count = 1;


    while(j<numsSize){
        if(nums[j] == nums[i]){
            j+=1;
        }
        else{
            nums[i+1]=nums[j];
            i+=1;
            j+=1;
            count += 1;
            
        }

    }
    return count;
}